chain/client/src/info.rs

```diff
@@ -50,14 +50,14 @@ pub struct InfoHelper {
     /// Sign telemetry with block producer key if available.
     validator_signer: Option<Arc<dyn ValidatorSigner>>,
     /// Telemetry actor.
-    telemetry_actor: Addr<TelemetryActor>,
+    telemetry_actor: Option<Addr<TelemetryActor>>,
```

Comment:
I do not understand this change.  Why are we storing this inside a `Option` now?  It seems that this is always set to `Some`.

Reply:
For testing.
Creating an `InfoHelper` in a test requires starting actix otherwise.

Reply:
I see.  Can we add that in the comments then please?  I see that as tech debt that we should resolve at some point.  In the meantime, we can at least help the readers by highlighting this.

Reply:
Better yet, can you create an issue for this please?

Reply:
Added comments explaining why this field is an `Option`.
A few alternatives that come to mind introduce prod-only and test-only code, which I'd like to avoid. The solution with `Option` is the simplest and I'd like to keep it.